facebook/__init__.py

```diff
@@ -298,18 +313,39 @@ def _add_to_post_args_or_args(arg_name, arg_value):
             _add_to_post_args_or_args("appsecret_proof", self.app_secret_hmac)
 
         try:
+            method = method or "GET"
+            url = FACEBOOK_GRAPH_URL + path
+
+            if self.logger and self.verbosity.value >= LoggingVerbosity.BASIC_REQUEST_INFO.value:
+                self.logger.debug('Request method: %s' % method)
+                self.logger.debug('Request URL: %s' % url)
+                self.logger.debug('Request Args: %s' % str(args))
+                self.logger.debug('Request Post Args: %s' % str(post_args))
```

Comment:
You should use `.format()` for formatting strings instead of `%`.

Reply:
`.format()` added in https://github.com/mobolic/facebook-sdk/pull/463/commits/b11fe55561d078fd3e3f91b9b09c92fc4f99e611